packages/flutter/lib/src/cupertino/list_tile.dart

```diff
@@ -376,13 +376,12 @@ class _CupertinoListTileState extends State<CupertinoListTile> {
           () => setState(() {
             _tapped = false;
           }),
+      onTapUp:
```

Comment:
Since onTapUp is called just before onTap, would moving the setState to before the widget.onTap inside onTap work the same?